src/htmx.js

```diff
@@ -799,7 +799,7 @@ return (function () {
          * @returns
          */
         function oobSwap(oobValue, oobElement, settleInfo) {
-            var selector = "#" + getRawAttribute(oobElement, "id");
+            var selector = '[id="' + getRawAttribute(oobElement, "id") + '"]';
```

Comment:
Wouldn't be nicer to use CSS.escape()?

```javasctipt
let selector = '#' + CSS.escape(getRawAttribute(oobElement, 'id'))
```

Reply:
Yes it would be nicer...but htmx maintains support for Internet Explorer 11 so `CSS.escape` (and any other ES6) isn't supported.

Reply:
hmm I am curious how this fix will be propagated to v2... Contribution rules says PR only into dev.

Reply:
Err, this is into bigskysoftware:dev